api/src/main/java/org/semanticweb/owlapi/model/parameters/ConfigurationOptions.java

```diff
@@ -97,7 +97,13 @@
     ENTITY_EXPANSION_LIMIT              ("100000000"),
     /** Repair illegal punnings 
      * automatically. */
-    REPAIR_ILLEGAL_PUNNINGS             (Boolean.TRUE);
+    REPAIR_ILLEGAL_PUNNINGS             (Boolean.TRUE),
+	/** True if http authorization 
+     * should be used. */
+    ACCEPT_HTTP_AUTHORIZATION             (Boolean.FALSE),
```

Comment:
Remove this and replace any references to it with equivalent references to ``AUTHORIZATION_VALUE``

Reply:
Removed  ACCEPT_HTTP_AUTHORIZATION and all the reference